### PR TITLE
Guide for listing coin on desktop

### PR DESCRIPTION
**Pull request by smk762**
ref: https://github.com/KomodoPlatform/developer-docs/issues/337

Docs for coins repo side of things will be added in a seperate PR.

### CODE REVIEW

File: docs/basic-docs/atomicdex/atomicdex-desktop/add-coin-to-atomicdex-desktop.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+## Update coins.json 
+
+| parameter              | Type     | Description           |
+| ---------------------- | -------- | --------------------- |
+| coin                   | string   | Ticker of the coin. Must match the "coin" field for an entry in the [coins repository](https://github.com/KomodoPlatform/coins/blob/master/coins#L5138) If the coin is on more than one network, use COIN-NETWORK as ticker, eg BTC-BEP20. |
+| name                   | string   | Full name of the coin. Should match the "fname" field for an entry in the [coins repository](https://github.com/KomodoPlatform/coins/blob/master/coins#L5138). Coins on more than one network share the same name. |
+| type                   | string   | Defines the category or network the coin is from. For example, `ERC20` for tokens on the Ethereum network; `Smart Chain` for Komodo Platform Antara smart chains; `UTXO` for Bitcoin forks (LTC, DOGE etc) |
+| asset                  | string   | (required for Smart Chains) Should match the "asset" value in the [coins repository](https://github.com/KomodoPlatform/coins/blob/master/coins#L7342) |
```
Comment (cipig) [inline docs/basic-docs/atomicdex/atomicdex-desktop/add-coin-to-atomicdex-desktop.md:27]: this is not needed in Desktop config, only https://github.com/cipig/coins/blob/master/coins has this field
ADEX Desktop only knows the tickers, like in TKL
```
  "TKL": {
    "coin": "TKL".
    "type": "Smart Chain",
```
and `"type": "Smart Chain"` identifies this entry as a KMD smart chain

Comment (smk762) [reply docs/basic-docs/atomicdex/atomicdex-desktop/add-coin-to-atomicdex-desktop.md:27]: @Milerius can you please confirm which of these fields are not required in https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/assets/config/0.5.4-coins.json#L4988-L4991 ?

I can's see `type` in https://github.com/KomodoPlatform/coins/blob/master/coins#L6653-L6665 so assume that is needed. 

`asset` only appears in smart chains for RICK/MORTY, so I assume this is not needed.

`name` appears to correspond with `fname` in coins repo. 

`coin` seems like unneeded duplication of the object key (though it links to `coin` in coins repo)

Comment (smk762) [reply docs/basic-docs/atomicdex/atomicdex-desktop/add-coin-to-atomicdex-desktop.md:27]: With local testing, removing `asset` is safe, but removing any of the other fields results in a crash on login.

File: docs/basic-docs/atomicdex/atomicdex-desktop/add-coin-to-atomicdex-desktop.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
+# Adding a Coin on AtomicDEX Desktop
+
+Before a coin can be listed in AtomicDEX, it needs to be confirmed for compatibility, and present within the [Komodo Platform coins repository](https://github.com/KomodoPlatform/coins).
+
+This involves:
+- Setting up two operational [electrum servers](https://electrumx.readthedocs.io/en/latest/)
```
Comment (cipig) [inline docs/basic-docs/atomicdex/atomicdex-desktop/add-coin-to-atomicdex-desktop.md:6]: Please change the link to https://electrumx-spesmilo.readthedocs.io/en/latest/
this one points to the old repo, the new one is https://github.com/spesmilo/electrumx

Comment (smk762) [reply docs/basic-docs/atomicdex/atomicdex-desktop/add-coin-to-atomicdex-desktop.md]: done

**Review by cipig — APPROVED** [review]

**Comment by smk762** [comment]
@gcharang This doc will go under a new section for AtomicDEX Desktop. I've just updated `atomicDEX-sidebar.js` and `navbar.js` could you please confirm all is as it should be?

**Review by tonymorony — APPROVED** [review]

**Pull request merged**